static/app/views/issueDetails/streamline/hooks/useSuspectFlags.tsx

```diff
@@ -28,7 +28,7 @@ export default function useSuspectFlags({
 
   // map flag data to arrays of flag names
   const auditLogFlagNames = hydratedFlagData.map(f => f.name);
-  const evaluatedFlagNames = event?.contexts.flags?.values.map(f => f.flag);
+  const evaluatedFlagNames = event?.contexts?.flags?.values?.map(f => f.flag);
```

Comment:
```suggestion
  const evaluatedFlagNames = event?.contexts?.flags?.values?.map(f => f.flag) ?? [];
```
`intersection()` takes array args

Reply:
<img width="766" alt="SCR-20241203-jusk" src="https://github.com/user-attachments/assets/dca64ca0-4f28-4cb7-b960-661cd9d168e7">
seems to be fine if `undefined` is passed in